packages/shopify-checkout/__tests__/mocks/index.ts

```diff
@@ -22,12 +22,19 @@ const checkoutUuids = {
   beginsWithNumber: '998877?key=123123'
 };
 export const checkoutIds = {
-  beginsWithLetter: Buffer.from(
-    'gid://shopify/Checkout/' + checkoutUuids.beginsWithLetter
-  ).toString('base64'),
-  beginsWithNumber: Buffer.from(
-    'gid://shopify/Checkout/' + checkoutUuids.beginsWithNumber
-  ).toString('base64')
+  encoded: {
+    beginsWithLetter: Buffer.from(
+      'gid://shopify/Checkout/' + checkoutUuids.beginsWithLetter
```

Comment:
Not something we _need_ to fix in this PR, but we might want to consider using template literals here instead of concatenation. I usually find them cleaner/easier to read.

Reply:
No problem at all, added in https://github.com/getnacelle/nacelle-js/pull/292/commits/5dbe9a252c96cced6660dfa4f3bc1ed0a30bbe99 :)